simulation-ros/src/README.md

```diff
@@ -0,0 +1,18 @@
+# The Src folder
```

Comment:
@albertohata and @EnyuCao do you think this file is necessary? I don't know if I got it wrong, but this looks like @EnyuCao 's note.

Reply:
Hi @jramadeu,please feel free to remove anything that you feel unnecessary. I am new to this project and it seems I always add comments with too much details.

Reply:
@EnyuCao having more comments than less is always better. Thank you!

Reply:
@jramadeu I was already planning to remove this file, as Andrew commented that before :) @EnyuCao, do this branch come from simulation-ros-trial or simulation-ros branch?

Reply:
@albertohata This branch was created from simulation-ros-trial. But I merged it with my previous branch, which was created from simulation-ros. During merging, I manually solved the conflicts to choose files from simulation-ros-trial. I hope the branch is not polluted.

Reply:
I found that I should not merge branches. So now I will create a new branch from simulation-ros-trial. Manually copy my files to this new branch and then it will be fine. @albertohata Is this OK?